homotopy-web/src/model/proof/generators.rs

```diff
@@ -38,11 +44,38 @@ impl GeneratorStyle for GeneratorInfo {
     fn shape(&self) -> Option<style::VertexShape> {
         Some(self.shape.clone().into())
     }
+
+    fn color_point(&self) -> Self::Color {
+        self.color.clone()
+    }
+
+    fn color_wire(&self) -> Self::Color {
+        Color((self.color.into_linear().lighten(0.05)).into())
+    }
+
+    fn color_surface(&self) -> Self::Color {
+        Color((self.color.into_linear().lighten(0.1)).into())
+    }
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct Color(pub Srgb<u8>);
 
+impl RenderableColor for Color {
```

Comment:
Related to the previous comments, these should really live in the svg/tikz modules in `homotopy_graphics`. And I don't think we need the trait.

Reply:
Yup, that sounds wise. I was trying to keep `palette` from becoming a dependency of `homotopy_graphics` but in heinsight I can probably keep all `palette` related code in `homotopy_graphics` instead.

So I'll try removing `palette` from `homotopy-web` dependencies and adding it to `homotopy_graphics` dependencies.

Reply:
This should clear up the comments above

Reply:
Yes, the `palette` dependency can move to `homotopy_graphics`.